python/res/enkf/enkf_simulation_runner.py

```diff
@@ -26,7 +31,43 @@ def _enkf_main(self):
 
     def runSimpleStep(self, job_queue, run_context):
         """ @rtype: int """
-        return self._run_simple_step(job_queue, run_context )
+        #### run simplestep ####
+        self._enkf_main().initRun(run_context)
+
+        if run_context.get_step():
+            ecl_config = self._enkf_main().ecl_config.assert_restart()
+
+        #### deselect load and parent failure #####
+        iactive = run_context.get_mask()
+
+        run_context.get_sim_fs().getStateMap().deselectMatching( iactive,
+            RealizationStateEnum.STATE_LOAD_FAILURE | RealizationStateEnum.STATE_PARENT_FAILURE)
+        
+        #### start queue ####
+        self.start_queue(run_context, job_queue)
+
+        #### deactivate failed realizations ####
+        totalOk = 0
+        totalFailed = 0
+        for i in range(len(run_context)):
+            if run_context.is_active(i):
+                run_arg = run_context[i]
+                if run_arg.run_status == RunStatusType.JOB_LOAD_FAILURE or run_arg.run_status == RunStatusType.JOB_RUN_FAILURE:
```

Comment:
In `job_queue_print_summary` `JOB_QUEUE_IS_KILLED` is regarded as a failure. I think we should do the same here.